remapping_commands.sh

```diff
@@ -131,7 +131,7 @@ echo '------------------------------3) Data extraction--------------------------
 # Extract chromosome, rsID, position on new genome, original variant allele, qual, filter and info fields, also 
 # creates old_ref_alleles.txt
 cd "$TMPDIR"
-samtools sort reads_aligned.bam -o reads_aligned.sorted.bam
+samtools sort reads_aligned_test.bam -o reads_aligned.sorted.bam
```

Comment:
Unless I misunderstand, isn't this file (`remapping_commands.sh`) the original pipeline which was rewritten into Nextflow and is no longer required? Couldn't we just delete it?

Reply:
I must have changed it inadvertently when refactoring. I guess it can be deleted completely.